urlCutover.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+school=$(basename "$(pwd)" | grep -o '[^-]*$') 
+env=$1
+oldUrl=$2
+newUrl=$3
+
+
+findAndReplace() {
+    # If the env is QA, then fileName is prefixed with qa 
+    filenamePrefix=""
+    if [ "$env" == "qa" ]; then
+        filenamePrefix="qa-"
+    fi
+
+    find . -type f -name "$filenamePrefix*.json" -exec sed -i "s/$oldUrl/$newUrl/g" {} +
```

Comment:
I've only run this in my mind, but if `filenamePrefix` is `""` (because we're cutting over a prod server), wouldn't the resulting `-name "*.json"` arg also match `qa-something.json`?